Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.5.1
+
+RUN mkdir /root-infrastructure
```

Comment:
Should this be root-ruby-style instead?

Reply:
🤦‍♂️ Yeah... That's what I get for copy/pasting